data/json/items/generic.json

```diff
@@ -2929,6 +2929,67 @@
     "weight": 11339,
     "volume": 12
   },
+  {
```

Comment:
I only gave your code a quick glance, but the changes in this file seem completely unrelated to adding the big bore rifles?

If they are unrelated, you should probably revert them and split them into a new pr.

Reply:
They're changes from my turret ammo drops PR. I don't know how they got into this PR or even why they appear as changes since that PR was already merged. Do I just delete those files from my PR?

Reply:
For future reference the way to deal with this is update your master and rebase your feature branch onto it.